client/components/action-panel/docs/example.jsx

```diff
@@ -41,6 +41,30 @@ const ActionPanelExample = () => (
 				</ActionPanelFooter>
 			</ActionPanel>
 		</div>
+		<div>
+			<ActionPanel>
+				<ActionPanelBody>
+					<ActionPanelFigure inlineBodyText={ true } align="left">
+						<img
+							src="/calypso/images/wordpress/logo-stars.svg"
+							width="170"
+							height="143"
+							alt="WordPress logo"
+						/>
+					</ActionPanelFigure>
+					<ActionPanelTitle>Action panel with left aligned image</ActionPanelTitle>
+					<p>
+						This is a description of the action. It gives a bit more detail and explains what we are
+						inviting the user to do.
+					</p>
+					<ActionPanelFooter>
+						<Button className="action-panel__support-button" href="/help/contact">
```

Comment:
I know there are classname lint errors from before, but it would be lovely if we could fix them in this PR.

Reply:
Yes, the lint errors seem to be because this file is called `example.jsx` but the class names don't conform to that component name. It would be great to fix that, I'll see if I can work out a tweak we can make to the rule.

Reply:
Would be great, but don't let that block the PR. I'm fine with this one as-is, since we already had those lint errors there.

Reply:
For now, I've disabled the rule in this file. Looking at the code for the rule, I think we could change it so it does something like ignore `docs/example.[j|t]sx` files or any components ending in `Example` because I think this might be a problem for a lot of the devdocs components. 

Either way, it's definitely something for another PR.